packages/paste-website/src/components/icons/TwilioLogo.tsx

```diff
@@ -13,7 +13,7 @@ const TwilioLogo = React.memo(({title = 'Twilio Logo', decorative = true, classN
   const titleId = useUID();
   return (
     <span style={{color, width: size, height: size, display: 'inline-block'}} className={className}>
-      <svg role="img" aria-hidden={decorative} width="100%" height="100%" aria-labelledby={titleId}>
+      <svg role="img" aria-hidden={decorative} width="100%" height="100%" viewBox="0 0 20 10" aria-labelledby={titleId}>
```

Comment:
thought: oof `0 0 20 10`. That's a little odd.

Reply:
There was a fair bit of trial and error to get it to look like it does currently and these values were the ones I found that work 🤷  . Maybe I'll get better intel from design when they have a look.

Reply:
Where is this being used? Should we use the Paste component instead?

Reply:
Nvm it's the logo + logomark. It's cut off on the website

<img width="241" alt="Screen Shot 2022-04-13 at 1 06 47 PM" src="https://user-images.githubusercontent.com/3821840/163261625-c86059e6-0a70-4a72-b401-992c08ac211a.png">

Reply:
@serifluous do you have the size of this logo anywhere? I'm just haphazardly plugging numbers into viewBox at this point and could use some guidance!

Reply:
You may get a quicker response if you send a designer the asset in Slack directly.

Reply:
This has been addressed!